src/hotspot/share/opto/memnode.cpp

```diff
@@ -2461,6 +2455,7 @@ const Type* LoadNode::klass_value_common(PhaseGVN* phase) const {
           // a primitive Class (e.g., int.class) has null for a klass field
           return TypePtr::NULL_PTR;
         }
+        // ???
```

Comment:
I suspect that we still need this code to support intrinsics like LibraryCallKit::inline_native_classID() and maybe other users of this field, but the comment below no longer makes sense.

Reply:
Thank you for noticing the ??? that I left in and the comment.